docs/about.rst

```diff
@@ -0,0 +1,57 @@
+**About**
```

Comment:
This appears to need a proper heading or title attribute in relation to the rest of the document, rather than just strong emphasis. An alternative may be to just remove this line altogether.